src/hooks/bin/configure

```diff
@@ -16,6 +16,33 @@
 . $SNAP/utilities/php-utilities
 . $SNAP/utilities/hook-utilities
 
+handle_apache_listen_address_config()
+{
+    listen_address="$(apache_listen_address)"
+    
+    # If no Listen address is submitted, then nothing to be done
+    if [ -z "$listen_address" ]; then
```

Comment:
Please save the default, and also save old values. Then compare against the old values, like everything else in here.